acme.sh

```diff
@@ -3613,7 +3617,11 @@ _ns_purge_cf() {
   _cf_d="$1"
   _cf_d_type="$2"
   _debug "Cloudflare purge $_cf_d_type record for domain $_cf_d"
-  _cf_purl="https://1.1.1.1/api/v1/purge?domain=$_cf_d&type=$_cf_d_type"
+  if [ "$CLOUDFLARE_DOHJSON_URL" != "" ]; then
+    _cf_purl="$ONE_ONE_ONE_ONE_URL?domain=$_cf_d&type=$_cf_d_type"
```

Comment:
what is the value of ONE_ONE_ONE_ONE_URL ?

Reply:
Replaced it with CLOUDFLARE_DOHJSON_URL to make it more explicit.

For networks where 1.1.1.1 is unusable, set it to `1001.cloudflare-dns.com` (see #2330)